lib/ruby-stackoverflow/configuration.rb

```diff
@@ -1,7 +1,7 @@
 
 module RubyStackoverflow
   class Configuration
-    attr_accessor :client_id,  :client_sceret , :client_key, :access_token, :api_url
+    attr_accessor :client_id,  :client_secret , :client_key, :access_token, :api_url
```

Comment:
Line is too long. [84/80]<br>Unnecessary spacing detected.<br>Space found before comma.